mongodbatlas/cloud_provider_snapshot_restore_jobs.go

```diff
@@ -20,7 +20,7 @@ type CloudProviderSnapshotRestoreJobsService interface {
 	Delete(context.Context, *SnapshotReqPathParameters) (*Response, error)
 }
 
-//CloudProviderSnapshotRestoreJobsServiceOp handles communication with the CloudProviderSnapshotRestoreJobs related methos of the
+//CloudProviderSnapshotRestoreJobsServiceOp handles communication with the CloudProviderSnapshotRestoreJobs related methods of the
```

Comment:
This threw me off for a second so making a comment for others - this PR is for continuous backup support but includes two (appreciated) comments regarding cloud provider snapshots, which is a different endpoint.